docs/en/table_design/table_types/primary_key_table.md

```diff
@@ -1,159 +1,212 @@
+
+:::note
 
-> **NOTE**
->
-> - In versions earlier than v3.0, the Primary Key table does not support decoupling the primary key and sort key.
-> - Since version 3.1, StarRocks's shared-data mode supports the Primary Key tables. Since version 3.1.4, Primary Key tables created in StarRocks shared-data clusters further support index persistence onto local disks.
+- Since v3.0, the sort key of a Primary Key table is decoupled from the table's primary key, and the sort key can be specified seperately. As such, table creation flexibility is improved.
+- Since v3.1, StarRocks's shared-data mode supports Primary Key tables. Since v3.1.4, Primary Key tables created in StarRocks shared-data clusters further support index persistence onto local disks.
+
+:::
 
 ## Scenarios
 
-- The Primary Key table is suitable for the following scenarios in which data needs to be frequently updated in real time:
-  - **Stream data in real time from transaction processing systems into StarRocks.** In normal cases, transaction processing systems involve a large number of update and delete operations in addition to insert operations. If you need to synchronize data from a transaction processing system to StarRocks, we recommend that you create a table that uses the Primary Key table. Then, you can use tools, such as CDC Connectors for Apache Flink®, to synchronize the binary logs of the transaction processing system to StarRocks. StarRocks uses the binary logs to add, delete, and update the data in the table in real time. This simplifies data synchronization and delivers 3 to 10 times higher query performance than when a Merge on Read (MoR) table of the Unique Key table is used. For example, you can use flink-connector-starrocks to load data. For more information, see [Load data by using flink-connector-starrocks](../../loading/Flink-connector-starrocks.md).
+The Primary Key table can support real-time data updates while ensuring efficient query performance. It is suitable for the following scenarios:
+
+- **Stream data in real time from transaction processing systems into StarRocks.** In normal cases, transaction processing systems involve a large number of update and delete operations in addition to insert operations. If you need to synchronize data from a transaction processing system to StarRocks, we recommend that you create a table that uses the Primary Key table. Then, you can use tools, such as CDC Connectors for Apache Flink®, to synchronize the binary logs of the transaction processing system to StarRocks. StarRocks uses the binary logs to add, delete, and update the data in the table in real time. This simplifies data synchronization and delivers 3 to 10 times higher query performance than when a Merge on Read table of the Unique Key table is used. For example, you can use flink-connector-starrocks to load data. For more information, see [Realtime synchronization from MySQL](../../loading/Flink_cdc_load.md).
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/c02000fc-f11b-40cf-b317-6a9d294af55e)
I guess you mean "a Unique Key table that uses the Merge on Read mechanism is used"